templates/useroptions.pug

```diff
@@ -100,6 +100,10 @@ mixin us-chat
       +rcheckbox("us-sort-rank", "Sort userlist by rank")
       +rcheckbox("us-sort-afk", "Sort AFKers to bottom")
       .col-sm-4
+      .col-sm-8
+        p.text-info The following option only applies if you have permission to see IP addresses in the first place.
```

Comment:
This line shouldn't need to exist. There is a separate tab for options that are only applicable to moderators.

Reply:
Yeah that's a fair point. Let me move it over.